ledger-core/v2/virtual/integration/method_test.go

```diff
@@ -108,7 +108,7 @@ func Method_PrepareObject(ctx context.Context, server *utils.Server, class refer
 	}
 }
 
-func TestVirtual_Method_WithoutExecutor(t *testing.T) {
+func TestVirtual_Method_WithoutExecutor(t *testing.T) { // todo why the test have a name "without executor"?
```

Comment:
remove this todo question from PR. "WithoutExecutor" indicates that "executor of builtin contracts" is mocked.